### DISCUSSION TITLE
CohereForAI/aya model support

### DISCUSSION THREAD

**Discussion by SSeanin** [sgl-project/sglang#1147] [Q&A] [answered]
Hi everyone,

I'm interested in exploring the potential of using the CohereForAI/aya model with the SGLang project. Given that aya is based on command architectures, I'm curious to know if it's currently supported or if there are any plans for integration.

I believe that leveraging aya's capabilities within SGLang could potentially offer interesting possibilities. I'd appreciate any insights or feedback on this topic.

**Comment by merrymercy** [comment, marked as answer]
Contributions are welcome!
How to Support a New Model -> https://github.com/sgl-project/sglang/blob/main/docs/en/model_support.md
